packages/typo3-docs-theme/assets/sass/components/_tabs.scss

```diff
@@ -1,6 +1,21 @@
+.tabs {
```

Comment:
I am not convinced that the tabs content should be that prominent with the gray background:

![Bildschirmfoto vom 2024-03-04 18-51-12](https://github.com/TYPO3-Documentation/render-guides/assets/2566282/72e06187-525d-43c0-b07e-c60d0c0a8e78)

Reply:
Additionally, a margin bottom is missing.

Reply:
IMO this is better than the currently too much detached tabs list, but I agree on smaller tabs it looks hefty. Maybe if we can add the margin bottom and try out further ideas after that?

Reply:
The tabs also don't really convince me, but we can also solve them in a follow up

Reply:
lets make a  follow up for the tab styles?

Reply:
I think, just marking the surrounding border darker should be enough.